packages/react/src/components/Masthead/Masthead.js

```diff
@@ -189,7 +189,7 @@ const Masthead = ({
               <IbmLogo />
 
               <div className={`${prefix}--header__search ${hasPlatform}`}>
-                {navigation && (
+                {navigation && !DDS_MASTHEAD_L1 && (
```

Comment:
Something a bit more nuanced. Instead of checking if the feature flag exists, I think we should have a prop for `navigationL1` and see if that's defined.

Reply:
And the data from `navigationL1` should be the same of the `navigation`?